inc/admin/upgrader.php

```diff
@@ -320,7 +319,6 @@ function rocket_new_upgrade( $wp_rocket_version, $actual_version ) {
 
 	if ( version_compare( $actual_version, '2.11', '<' ) ) {
 		rocket_clean_minify();
-		rocket_clean_cache_busting();
```

Comment:
What is the impact of removing this for customers running this version or older? Do we want to keep this and let it run through the deprecated functions?

Reply:
Your comment makes me think we would want to run it one last time to clean the folder of the files that will no longer be used.

Reply:
- Clean busting directory on update: https://github.com/wp-media/wp-rocket/pull/2616/commits/352f2e22a77fd743dc78f959bd9f6567355f30af
- Undeprecate `rocket_clean_cache_busting()`: https://github.com/wp-media/wp-rocket/pull/2616/commits/c3860b6375c722dc87aa145f97df1539a5adf3f2